packages/playground/react-jsx-runtime/App.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react'
+import { css } from '@emotion/react'
+
+import _Switch from 'react-switch'
```

Comment:
This line is a bit weird.
The dist in `react-switch` is like the following:

```js
Object.defineProperties(exports, '__esModule', { value: true })
exports.default = ReactSwitch
```

When running with `yarn vite`, `react-switch` will be pre-bundled with esbuild and it will emit `export default xxx` like it is an esm;
but when run with `yarn vite build`, `react-switch` will be bundled with rollup and commonjs plugin, the result is `export default { default: ReactSwitch }`.

I think the way rollup/commonjs plugin works has a bug and it did not correctly handles `__esModule` flag.
This issue might be related: https://github.com/rollup/plugins/issues/939 and the [fix](https://github.com/rollup/plugins/commit/7757aebc74e8b40598e7f67e71dd8e427b0d0095) is included `19.0.2`. I tried this version but still did not correctly transform esModule flag.

Reply:
Yeah, there's a Vite issue for this too:
https://github.com/vitejs/vite/issues/2139#issuecomment-854960323

Reply:
ops, this is an infamous issue already. I think eventually this issue should be fixed on the rollup commonjs plugin. But not sure why it is still not fixed by 19.0.2

Reply:
@pengx17 commonjs@19.0.2 only fix code reassign to `module.exports`  with `__esModule` key like

```
module.exports = { __esModule = true , default } 
```

Reply:
Wow, thanks for the insights githoniel! This is very helpful!